nsis/lang/simpchinese.nsi

```diff
@@ -241,37 +241,37 @@ LangString str_msg_unregistering ${LANG_SIMPCHINESE} \
 LangString str_vimrc_page_title    ${LANG_SIMPCHINESE} \
     "设置 _vimrc"
 LangString str_vimrc_page_subtitle ${LANG_SIMPCHINESE} \
-    "选择键盘、鼠标和增强选项。"
+    "选择键盘、鼠标和扩展设置。"
 
 LangString str_msg_compat_title    ${LANG_SIMPCHINESE} \
     "Vi / Vim 行为"
 LangString str_msg_compat_desc     ${LANG_SIMPCHINESE} \
-    "&Compatibility and enhancements"
+    "兼容性与扩展(&B)"
 LangString str_msg_compat_vi       ${LANG_SIMPCHINESE} \
-    "Vi compatible"
+    "原始 Vi"
 LangString str_msg_compat_vim      ${LANG_SIMPCHINESE} \
-    "Vim original"
+    "原始 Vim"
 LangString str_msg_compat_defaults ${LANG_SIMPCHINESE} \
-    "Vim with some enhancements (load defaults.vim)"
+    "Vim 原始版本和部分扩展 (加载 defaults.vim)"
 LangString str_msg_compat_all      ${LANG_SIMPCHINESE} \
-    "Vim with all enhancements (load vimrc_example.vim) (Default)"
+    "Vim 原始版本和所有扩展 (加载 vimrc_example.vim) (默认)"
 
 LangString str_msg_keymap_title   ${LANG_SIMPCHINESE} \
     "键盘映射"
 LangString str_msg_keymap_desc    ${LANG_SIMPCHINESE} \
-    "&Remap a few keys for Windows (Ctrl-V, Ctrl-C, Ctrl-A, Ctrl-S, Ctrl-F, etc)"
+    "为 Windows 映射按键(&R) (例如:Ctrl-V, Ctrl-C, Ctrl-A, Ctrl-S, Ctrl-F 等)"
 LangString str_msg_keymap_default ${LANG_SIMPCHINESE} \
-    "Do not remap keys (Default)"
+    "不映射按键 (默认)"
 LangString str_msg_keymap_windows ${LANG_SIMPCHINESE} \
-    "Remap a few keys"
+    "映射一些按键"
 
 LangString str_msg_mouse_title   ${LANG_SIMPCHINESE} \
     "鼠标"
 LangString str_msg_mouse_desc    ${LANG_SIMPCHINESE} \
-    "&Behavior of right and left buttons"
+    "左键和右键行为(&B)"
 LangString str_msg_mouse_default ${LANG_SIMPCHINESE} \
-    "Right: popup menu, Left: visual mode (Default)"
+    "右键：弹出菜单, 左键：可视化模式 (Unix)"
```

Comment:
Shouldn't "Unix" be "Default"?
The "Default" setting is differ from "Unix".

Reply:
This is my mistake, I will fix it.